app/src/androidTest/java/org/mozilla/fenix/ui/SettingsAddonsTest.kt

```diff
@@ -66,7 +66,10 @@ class SettingsAddonsTest {
             verifyAddons()
         }.openAddonsManagerMenu {
             addonsListIdlingResource =
-                RecyclerViewIdlingResource(activityTestRule.activity.findViewById(R.id.add_ons_list), 1)
+                RecyclerViewIdlingResource(
+                    activityTestRule.activity.findViewById(R.id.add_ons_list),
+                    1
+                )
```

Comment:
Wondering if we need theses changes since it's different from the intended license change?

Reply:
No intentional change. I assume intellij did "something". I'm going to fix it.

Reply:
Fixed